src/modules/tools/temperaturecontrol/TemperatureControl.h

```diff
@@ -83,14 +81,17 @@ class TemperatureControl : public Module {
         float PIDdt;
 
         // Temperature runaway values
+        enum RUNAWAY_TYPE {NOT_HEATING, HEATING_UP, COOLING_DOWN, MAINTAINING_TEMPERATURE};
+
         RUNAWAY_TYPE runaway_state;      
+        uint8_t runaway_heating_timer;
+        float runaway_last_temperature;
```

Comment:
That is used to handle the case where the temperature is being increased or decreased to attain the target.